.github/workflows/test.yml

```diff
@@ -84,12 +90,49 @@ jobs:
         if: github.event_name != 'schedule'
         run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
 
-  # This job reports the results of the test matrix above
+  doc-checks:
+    name: Test documentation website
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Set up Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: website/package-lock.json
+      - name: Install local website package
+        working-directory: ./website
+        run: npm ci
+      - name: Ensure website builds
+        working-directory: ./website
+        run: npm run build
+
+  # This job reports the results of the test jobs above and is used to enforce status checks in
+  # the repo settings without needing to update those settings everytime the test jobs are updated.
   test:
     if: always()
-    needs: test-matrix
     runs-on: ubuntu-latest
+    needs: [test-matrix, deno-checks, shellcheck, doc-checks]
```

Comment:
Was the old job broken? Deno checks failed the build correctly even though it wasn't explicitly listed here or in the conditional.

Reply:
It has to do with branch protection rules. The current ones for this repo have enabled the setting to enforce specific status checks before allowing a merge to `main`:

<img width="780" alt="image" src="https://github.com/phylum-dev/cli/assets/18729796/24d9deae-98f2-4fd6-b02f-4838fbe57823">

These are the same status checks that have a `required` label by them on the conversation view of the PR:

<img width="855" alt="image" src="https://github.com/phylum-dev/cli/assets/18729796/4e687610-9332-434c-a840-f1d59c40c599">

The old job was not broken in the sense that the overall build would show a failure if one of the status checks failed. It was just that it was still possible to merge the PR to `main` if one of the failed status checks was not `required`.

With that in mind, the initial idea for a "rollup" test report job like this to exist in the first place was to reduce the amount of coordination required when adding or modifying jobs in the test workflow. That is, it is much easier to update the rollup job to add new test jobs than it is to add a new job (or change an existing job's name) and then _also_ remember to update the repo settings...assuming permissions exist to make that change.